x/feegrant/keeper/keeper.go

```diff
@@ -237,7 +237,7 @@ func (k Keeper) UseGrantedFees(ctx context.Context, granter, grantee sdk.AccAddr
 
 	if remove {
 		// Ignoring the `revokeFeeAllowance` error, because the user has enough grants to perform this transaction.
-		k.revokeAllowance(ctx, granter, grantee)
+		err = k.revokeAllowance(ctx, granter, grantee)
```

Comment:
this shadow the `grant.Accept` error. The command above says the error should be ignored.